charts/agent/README.md

```diff
@@ -329,6 +329,12 @@ Chart to install K8s collection stack based on Observe Agent
 | node-logs-metrics.serviceAccount.create | bool | `false` |  |
 | node-logs-metrics.serviceAccount.name | string | `"observe-agent-service-account"` |  |
 | node.containers.logs.enabled | bool | `true` |  |
+| node.containers.logs.exclude | string | `"[]"` |  |
+| node.containers.logs.excludeOlderThan | string | `"1d"` |  |
```

Comment:
maybe this could be called "lookback period" or something? is there a term for that interval?

Reply:
done